src/components/RightPane/Map/UCIMap.tsx

```diff
@@ -1,30 +1,32 @@
 import React, { PureComponent } from 'react';
 import { Map, TileLayer, withLeaflet, Polyline, Marker, LeafletContext } from 'react-leaflet';
-import { Building } from './static/building';
+import Building from './static/building';
 import buildingCatalogue from './static/buildingCatalogue';
 import locations from '../SectionTable/static/locations.json';
 import AppStore from '../../../stores/AppStore';
 import MapMenu from './MapMenu';
 import MapMarker from './MapMarker';
-import Locate from 'leaflet.locatecontrol';
-import Leaflet, { LeafletMouseEvent } from 'leaflet';
+import Leaflet, { Control, LeafletMouseEvent } from 'leaflet';
 import analyticsEnum, { logAnalytics } from '../../../analytics';
 import { CalendarEvent, CourseEvent } from '../../Calendar/CourseCalendarEvent';
 import { Coord, MapBoxResponse } from './static/mapbox'
+import '../../../../node_modules/leaflet.locatecontrol/dist/L.Control.Locate.min.js';
```

Comment:
Yeah let's just leave it hacky for now. If it breaks later we can just revert to the TS-ignore fix.